core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -30,8 +30,22 @@ public interface Configs {
 
   String ENTITY_KV_ROCKSDB_BACKEND_PATH_KEY = "gravitino.entity.store.kv.rocksdbPath";
 
-  Long DEFAULT_KV_DELETE_AFTER_TIME = 604800000L; // 7 days
-  String KV_DELETE_AFTER_TIME_KEY = "gravitino.entity.store.kv.deleteAfterTimeMs";
+  // Config for data keep time after soft deletion, in milliseconds.
+  String STORE_DELETE_AFTER_TIME_KEY = "gravitino.entity.store.deleteAfterTimeMs";
+  Long DEFAULT_STORE_DELETE_AFTER_TIME = 604800000L; // 7 days
+  // The maximum allowed keep time for data deletion, in milliseconds. Equivalent to 30 days.
+  Long MAX_DELETE_TIME_ALLOW = 1000 * 60 * 60 * 24 * 30L;
```

Comment:
Maybe we don't need to restrict users from setting the maximum value.

Reply:
> Maybe we don't need to restrict users from setting the maximum value.

For deletion, I think it is necessary to limit the maximum allowed keep time.